app/hotels/src/singleHotel/hotelInformation/description/Description.js

```diff
@@ -37,7 +37,7 @@ export class Description extends React.Component<Props> {
           truncatedText={
             <Translation id="single_hotel.description.show_more" />
           }
-          revealedText={<Translation id="single_hotel.description.show_more" />}
+          revealedText={<Translation id="single_hotel.description.show_less" />}
```

Comment:
Hm? Nobody ever noticed this but wrong shade of the smallest grey line are repeatately reported? 😄